dev/ci/ci-autosubst-ocaml.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download autosubst-ocaml
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+export COQEXTRAFLAGS='-native-compiler no'
+( cd "${CI_BUILD_DIR}/autosubst-ocaml"
+  dune build
```

Comment:
needs to be something like 
```suggestion
  dune build --root . --only-packages=(your dune package name) @install
```

Reply:
I don't manage to make anything with `--only-packages`	work here, because whatever I do I get `Error: I don't know about package (package_name)` (trying `autosubst`, `autosubst_lib`, `autosubst-ocaml`, `autosubst_ocaml`) and changing the `--root` to `lib`.

I looked around what other projects do and found the approach with `-p`. I don't know whether this is right, but locally it works -- see the new commit.